spec/services/notification_handler_service_spec.rb

```diff
@@ -6,6 +6,7 @@
 
   after do
     Timecop.return
+    Sidekiq::Testing.inline!
```

Comment:
Rather than a before and after could we put this all in an around block. Then we could do `Sidekiq::Testing.fake! { example.run }` (only learnt from this pr that it accepts a block 🎉 ) which would mean we wouldn't need to be in sync with rails_helper ?

Reply:
Good idea - how does 5993053 look?